test/integration/cluster/test_cluster_demote.py

```diff
@@ -133,18 +133,21 @@ def test_cluster_demote_auth(tt_cmd, tmpdir_with_cfg, etcd, auth):
 
         if auth == "url":
             env = None
-            url = f"http://{etcd_username}:{etcd_password}@{etcd.host}:{etcd.port}/prefix?timeout=5"
+            url = (
+                f"http://{DEFAULT_USERNAME}:{DEFAULT_PASSWORD}@"
+                f"{etcd.host}:{etcd.port}/prefix?timeout=5"
+            )
             demote_cmd = [tt_cmd, "cluster", "rs", "demote", "-f", url, "instance-001"]
         elif auth == "flag":
             env = None
             url = f"{etcd.endpoint}/prefix?timeout=5"
             demote_cmd = [tt_cmd, "cluster", "rs", "demote", "-f",
-                          "-u", etcd_username,
-                          "-p", etcd_password,
+                          "-u", DEFAULT_USERNAME,
+                          "-p", DEFAULT_PASSWORD,
```

Comment:
It still looks confusing, it would be more clear:
```suggestion
                          "-u", etcd_helper.DEFAULT_USERNAME,
                          "-p", etcd_helper.DEFAULT_PASSWORD,
```
or
```suggestion
                          "-u", ETCD_DEFAULT_USERNAME,
                          "-p", ETCD_DEFAULT_PASSWORD,
```